tools/run_tests/run_interop_tests.py

```diff
@@ -1071,6 +1083,12 @@ def aggregate_http2_results(stdout):
     action='store_const',
     const=True,
     help='Run cloud_to_prod_auth tests.')
+argp.add_argument(
```

Comment:
looks like the on_gce argument is only applicable to the newly added "google_default_credentials" test case and is irrelevant for all other tests cases?

Reply:
Correct. I wasn't totally sure about this, but `on_gce` seemed simplest. Can rename possibly.